Microservices guide general update to 3.1 (#16202)
dotnet/docs: docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

> ---

Reviewer: could be good to blur out your e-mail on the previous image. Also, not sure if that red lettering with white border has good contrast

Author: Done!

> title: Orchestrating microservices and multi-container applications for high scalability and availability

Reviewer: not related to your PR but we've been removing ing from titles 

```suggestion
title: Orchestrate microservices and multi-container applications for high scalability and availability
```

description: Discover the options to orchestrate microservices and multi-container applications for high scalability and availability and the possibilities of Azure Dev Spaces while developing Kubernetes application lifecycle.
ms.date: 01/30/2020
---

> # Orchestrating microservices and multi-container applications for high scalability and availability

Reviewer: ```suggestion
# Orchestrate microservices and multi-container applications for high scalability and availability
```

There are no fees for any of the software installed by default as part of AKS. All default options are implemented with open-source software. AKS is available for multiple virtual machines in Azure. You're charged only for the compute instances you choose, as well as the other underlying infrastructure resources consumed, such as storage and networking. There are no incremental charges for AKS itself.

> The default production deployment option for Kubernetes nowadays, is to use Helm charts, introduced in the next section.

Reviewer: I don't think you need to say nowadays. How about?

```suggestion
The default production deployment option for Kubernetes is to use Helm charts, which will be introduced in the next section.
```

> ## Deploying with Helm charts into Kubernetes clusters

Reviewer: ```suggestion
## Deploy with Helm charts into Kubernetes clusters
```

Helm is maintained by the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) - in collaboration with Microsoft, Google, Bitnami and the Helm contributor community.

> For further implementation information on Helm charts and Kubernetes check the post on [Using Helm Charts to deploy eShopOnContainers to AKS](https://github.com/dotnet-architecture/eShopOnContainers/wiki/Deploy-to-Azure-Kubernetes-Service-(AKS)).

Reviewer: ```suggestion
For more implementation information on Helm charts and Kubernetes, see the [Using Helm Charts to deploy eShopOnContainers to AKS](https://github.com/dotnet-architecture/eShopOnContainers/wiki/Deploy-to-Azure-Kubernetes-Service-(AKS)) post.
```

This feature is based on URL prefixes, so when using any dev space prefix in the url, a request is served from the target microservice if it exists in the dev space, otherwise it's forwarded up to the first instance of the target microservice found in the hierarchy, eventually getting to the master dev space at the top.

> You can see the [eShopOnContainers wiki page on Azure Dev Spaces](https://github.com/dotnet-architecture/eShopOnContainers/wiki/Azure-Dev-Spaces), to get a practical view on a concrete example.

Reviewer: ```suggestion
To get a practical view on a concrete example, see the [eShopOnContainers wiki page on Azure Dev Spaces](https://github.com/dotnet-architecture/eShopOnContainers/wiki/Azure-Dev-Spaces).
```
